src/pbview.cpp

```diff
@@ -367,13 +380,17 @@ std::string PbView::format_line(const std::string& podlist_format,
 {
 	FmtStrFormatter fmt;
 
+	const double speed_kbps = dl.kbps();
+	const auto speed = get_speed_human_readable(speed_kbps);
+
 	fmt.register_fmt('i', strprintf::fmt("%u", pos + 1));
 	fmt.register_fmt('d',
 		strprintf::fmt("%.1f", dl.current_size() / (1024 * 1024)));
 	fmt.register_fmt(
 		't', strprintf::fmt("%.1f", dl.total_size() / (1024 * 1024)));
 	fmt.register_fmt('p', strprintf::fmt("%.1f", dl.percents_finished()));
-	fmt.register_fmt('k', strprintf::fmt("%.2f", dl.kbps()));
+	fmt.register_fmt('k', strprintf::fmt("%.2f %s", speed.first,
```

Comment:
I'd prefer to introduce a new specifier, `K`, so people's custom settings aren't broken, and those who like the old default can go back to it.

Reply:
Good idea

- [ ] Implement